tests/test_adaptor.hh

```diff
@@ -31,7 +31,6 @@
 #ifndef TESTS_TEST_ADAPTOR_HH_
 #define TESTS_TEST_ADAPTOR_HH_
 
-#include "tests.hh"
```

Comment:
Just out of curiosity: How do you check for unused includes?
Seems like a couple of them piled up...

Reply:
manually commenting out #include, and checking what compiles ...

There is a tool to do it systematically: https://github.com/include-what-you-use/include-what-you-use